packages/react-component-library/src/components/NumberInput/NumberInput.stories.tsx

```diff
@@ -18,24 +19,16 @@ const initialValues: Data = {
   gold: 1,
 }
 
-const onSubmit = (data: Data): void => {
-  console.log(`Form Submit ${JSON.stringify(data)}`)
-}
-
-const onCancel = () => {
-  console.log('Cancel')
-}
-
 const FormikNumberInput = withFormik(NumberInput)
 
 stories.add('Vanilla', () => (
   <div style={{ margin: 50 }}>
-    <Formik initialValues={initialValues} onSubmit={onSubmit}>
+    <Formik initialValues={initialValues} onSubmit={action('Form submit')}>
```

Comment:
maybe

`{ (data) => action(data) )` ?

Reply:
This outputs `data` as expected:

![Screenshot 2019-10-03 at 13 49 57](https://user-images.githubusercontent.com/56078793/66127961-c9a74e80-e5e4-11e9-8a43-bd396f1282a0.png)